src/purchases.ts

```diff
@@ -218,15 +245,29 @@ export default class Purchases {
       throw new Error("appUserID needs to be a string");
     }
 
+    let purchasesCompletedByToUse: PURCHASES_ARE_COMPLETED_BY_TYPE;
```

Comment:
We can probably default this to `PURCHASES_ARE_COMPLETED_BY_TYPE.REVENUECAT` and just change it in the case `isPurchasesAreCompletedByMyApp` is true?

Reply:
That works, and it gets rid of the else bracket after the following if. I like it!